edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -65,55 +65,9 @@ def authenticate(self, request):
         # .. custom_attribute_name: is_forgiving_jwt_cookies_enabled
         # .. custom_attribute_description: This is temporary custom attribute to show
         #      whether ENABLE_FORGIVING_JWT_COOKIES is toggled on or off.
+        #      See docs/decisions/0002-remove-use-jwt-cookie-header.rst
         set_custom_attribute('is_forgiving_jwt_cookies_enabled', is_forgiving_jwt_cookies_enabled)
 
-        # TODO: Robert: Refactor back into this single method in a separate commit
-        if is_forgiving_jwt_cookies_enabled:
-            return self._authenticate_forgiving_jwt_cookies(request)
-        return self._authenticate_original(request)
-
-    def _authenticate_original(self, request):
-        try:
-            user_and_auth = super().authenticate(request)
-
-            # Unauthenticated, CSRF validation not required
-            if not user_and_auth:
-                return user_and_auth
-
-            # Not using JWT cookies, CSRF validation not required
-            use_jwt_cookie_requested = request.META.get(USE_JWT_COOKIE_HEADER)
-            if not use_jwt_cookie_requested:
-                return user_and_auth
```

Comment:
**Note to reviewer:** The refactor makes a change to this original code (not seen in the changed code) where
we now check to see if there is JWT cookie, rather than checking if the `USE_JWT_COOKIE` header exists, to determine whether to check for CSRF protection.